source/User_Guide/email_activity_beta.md

```diff
@@ -10,9 +10,10 @@ seo:
   description:
 ---
 
-The Email Activity feed allows you to view a snapshot of email-related activity associated with your account and any subusers. Our latest enhancements to Email Activity give you increased visibility into the email delivery flow to help expedite the troubleshooting process.
+The Email Activity feed allows you to view a snapshot of email-related activity associated with your account and any subusers. Enhanced email activity and storage gives you increased visibility into the email delivery flow to help expedite the troubleshooting process.  
```

Comment:
maybe 'email delivery flow of each email message to expedite'... Would be great to highlight the fact that the new email activity feed shows events by email message vs. old email activity's log of disparate events that could be tied to the same message.

Reply:
Also are we sure subuser information is shown here? I'll confirm with Scott K.

Reply:
You are right, it is.